04_environment/index.tex

```diff
@@ -1 +1,400 @@
+
+All of the aforementioned behaviours are of course influenced by island tactics and therefore, an island programmed to be greedy will not cease to be greedy no matter how the surrounding islands’ behaviours change, unless the greedy island can adapt. Therefore, conclusions drawn on specific foraging methods, and how agents react with them, need to take this into account.\\
+
+\section{Disaster}
+\subsection{Disaster Background}
+
+A game has been designed, where agents are at risk, they will be directly confronting a risk/a disaster once every round. This subset of the game creates a dilemma, where each island (agent) has to survive. They will have the opportunity to work together by investing collectively to the common pool, the disaster will be mitigated if the islands invest enough resources to reach a desired threshold. Otherwise, the islands would face negative impact, which would diminish their available resources. It is worth mentioning that the disaster will first extract resources from the common pool, and then if not enough resources have been invested, resources will be depleted from each island proportionally to their island’s damage.\\
+
+Furthermore, the disaster has been designed to comprise of an epicentre (eye of the storm), which is limited within the archipelago bounds, it is represented using Cartesian coordinates (EpiX, EpiY). Therefore, all islands or a subset of islands may be affected due to their proximity to the disaster epicentre. The island’s proximity to the disaster epicentre is directly proportional to the island’s damage. In other words, the island's available resources will be diminished based on the disaster magnitude and its location with respect to each island’s fixed location.\\
+
+The following example can briefly explain the dilemma:\\
+
+The closest an island from the eye of the storm, the larger the severity on that island. Therefore, the more resources are depleted. Let’s say that the disaster epicentre hits around island 1 as shown in Figure below. and that the desired common pool threshold has not been met. Island 1 would be experiencing severe damages , and island 2 and 6 some damages and the other islands low to no damage. Therefore, Island 1,2 and 6 have the risk of not surviving if they are lacking resources, whereas island 3, 4, and 5 would not be affected as much by the disaster.
+
+\begin{figure}[!htb]
+    \centering
+    \includegraphics[width=1\textwidth]{04_environment/Images/Disaster eye of the storm severity.PNG}
+    \caption{Disaster Epicenter effects}
+    \label{Images:Disaster eye of the storm severity}
```

Comment:
wrong label for the figure.
should have been:
\label{**fig**:Disaster eye of the storm severity}